cost/flexera/cco/cheaper_regions/cheaper_regions.pt

```diff
@@ -8,30 +8,31 @@ category "Cost"
 tenancy "single"
 default_frequency "daily"
 info(
-  version: "2.3",
+  version: "2.4.0",
   provider: "Flexera",
   service: "Cloud Cost Optimization",
   policy_set: "Cloud Cost Optimization"
 )
 
-###############################################################################
-# Permissions
-###############################################################################
-
-
 ###############################################################################
 # Parameters
 ###############################################################################
 
 parameter "param_email" do
   type "list"
+  category "Policy Settings"
   label "Email addresses of the recipients you wish to notify"
+  description "A list of email addresses to notify."
+  # No default value, user input required
 end
 
 parameter "param_billing_centers" do
-  label "Billing Center Name"
   type "list"
+  category "Filters"
+  label "Billing Center Name"
+  description "List of billing centers to be filtered."
   min_length 1
+  # No default value, user input required
```

Comment:
This should only be a required parameter if Billing Center filtering is required. Is there a reason this policy can't just be run across their entire estate?

Reply:
There is already a logic that, in case that user doesn't enter any `billing_centers` then it will take the top ones (The ones that don't have a parent) so, I removed this comment on code.